test/specs/mainnet/getAddresses/arbitrum.spec.js

```diff
@@ -1,13 +1,16 @@
 import * as dotenv from 'dotenv';
 dotenv.config(); // init dotenv
 import { Factory, PrimeSdk } from '@etherspot/prime-sdk';
+import { EtherspotBundler } from 'ethers';
```

Comment:
Is this valid? Do we have an EtherspotBundler in ethers package?

Reply:
@ch4r10t33r Its new implementation is available in the examples.

Reply:
I dont think its right and I am surprised that they have such a thing if exists @Jineshdarjee

Reply:
ok, I'll remove it.

Reply:
Committed updated code